android/copy_charting_library_files.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Couldn't we hoist that file outside of a specific "technology" and just have a dedicated one that would import it and pass some specific params to it as an object/array?

Reply:
We could refactor this definitely. I didn't yet because it's just a helper script. It would probably be slightly easier to maintain if there were one copy.